### PR TITLE
Hide No Hit and Timing Target separately

### PR DESCRIPTION
**Pull request by Raoul1808**
Changed the Hide UI Elements tweak so it hides the Timing Target and No Fail icons independently from Otto.

This change also modified the translation as follows:
- Edited french and english translations for `HideUIElements.AUTO`
- Added 2 new keys in HideUIElements (`TIMING_TARGET` and `NO_FAIL_ICON`)

### CODE REVIEW

File: AdofaiTweaks/Tweaks/HideUiElements/HideUiElementsSettings.cs [hunk 1]
```diff
@@ -86,16 +92,21 @@ public class HideUiElementsSettings : TweakSettings
                 AdofaiTweaks.ReleaseNumber >= 94 ? null : scnEditor.instance);
 
             if (isEditingLevel) {
-                if (scnEditor.instance?.ottoCanvas.gameObject.activeSelf == hideOtto) {
-                    scnEditor.instance.ottoCanvas.gameObject.SetActive(!hideOtto);
+                scnEditor.instance.autoImage.enabled = !hideOtto;
```
Comment (PizzaLovers007) [inline AdofaiTweaks/Tweaks/HideUiElements/HideUiElementsSettings.cs:94]: I think this only hides the image, can you disable the button as well?

File: AdofaiTweaks/Tweaks/HideUiElements/HideUiElementsSettings.cs [hunk 2]
```diff
@@ -71,12 +72,17 @@ public class HideUiElementsSettings : TweakSettings
...
+            bool hideNoFail = hideEverything || selectedProfile.HideNoFailIcon;
             bool hideBeta = hideEverything || selectedProfile.HideBeta;
             bool hideTitle = hideEverything || selectedProfile.HideTitle;
 
             var tweakEnabled = AdofaiTweaks.IsEnabled && IsEnabled;
             hideEverything &= tweakEnabled;
             hideOtto &= tweakEnabled;
+            hideTimingTarget &= tweakEnabled;
+            hideNoFail &= tweakEnabled;
+            hideTitle &= tweakEnabled;
```
Comment (PizzaLovers007) [inline AdofaiTweaks/Tweaks/HideUiElements/HideUiElementsSettings.cs]: `hideTitle` is already set later in line 87, so you can delete this.

**Review by PizzaLovers007 — CHANGES_REQUESTED** [review]
Thanks for the changes!

File: AdofaiTweaks/Tweaks/HideUiElements/HideUiElementsSettings.cs (hunk 2 above)
Comment (Raoul1808) [reply AdofaiTweaks/Tweaks/HideUiElements/HideUiElementsSettings.cs]: My bad, I accidentally duplicated it when adding the 2 new options. I'm removing it

File: AdofaiTweaks/Tweaks/HideUiElements/HideUiElementsSettings.cs (hunk 1 above)
Comment (Raoul1808) [reply AdofaiTweaks/Tweaks/HideUiElements/HideUiElementsSettings.cs:94]: Done, the button was indeed still working

**Review by PizzaLovers007 — APPROVED** [review]
Thanks!

**Pull request merged**